Core/include/Acts/TrackFinding/CombinatorialKalmanFilter.hpp

```diff
@@ -1349,6 +1351,34 @@ class CombinatorialKalmanFilter {
 
     return std::move(combKalmanResult.collectedTracks);
   }
+
+  /// Combinatorial Kalman Filter implementation, calls the Kalman filter
+  ///
+  /// @tparam source_link_iterator_t Type of the source link iterator
+  /// @tparam start_parameters_t Type of the initial parameters
+  /// @tparam parameters_t Type of parameters used for local parameters
+  ///
+  /// @param initialParameters The initial track parameters
+  /// @param tfOptions CombinatorialKalmanFilterOptions steering the track
+  ///                  finding
+  /// @param trackContainer Input track container to use
```

Comment:
```suggestion
  /// @param trackContainer Track container in which to store the results
```
or something better. I was triggered by the word `Input` there on this non-`const` parameter.
If this is changed, it should also be changed at line 1238.